config/v1/stable.clusterversion.testsuite.yaml

```
@@ -14,3 +14,114 @@ tests:
       kind: ClusterVersion
       spec:
         clusterID: foo
+  - name: Should allow image to be set
+    initial: |
+      apiVersion: config.openshift.io/v1
+      kind: ClusterVersion
+      spec:
+        clusterID: foo
+        desiredUpdate:
+          image: bar
+    expected: |
+      apiVersion: config.openshift.io/v1
+      kind: ClusterVersion
+      spec:
+        clusterID: foo
+        desiredUpdate:
+          image: bar
+  - name: Should allow version to be set
+    initial: |
+      apiVersion: config.openshift.io/v1
+      kind: ClusterVersion
+      spec:
+        clusterID: foo
+        desiredUpdate:
+          version: 4.11.1
+    expected: |
+      apiVersion: config.openshift.io/v1
+      kind: ClusterVersion
+      spec:
+        clusterID: foo
+        desiredUpdate:
+          version: 4.11.1
+  - name: Should allow architecture to be empty
+    initial: |
+      apiVersion: config.openshift.io/v1
+      kind: ClusterVersion
+      spec:
+        clusterID: foo
+        desiredUpdate:
+          architecture:
```

I would prefer to see this explicitly with the empty string
```suggestion
          architecture: ""
```